src/tests/functional/shared_test_classes/src/single_op/conversion.cpp

```diff
@@ -46,6 +46,10 @@ void ConversionLayerTest::SetUp() {
     std::tie(conversion_type, shapes, input_type, convert_type, targetDevice) = GetParam();
     init_input_shapes(shapes);
 
+    if (convert_type == ov::element::f16) {
+        configuration.insert(ov::hint::inference_precision(ov::element::f16));
```

Comment:
Could you please elaborate why this infer precision is needed?

Reply:
Otherwise, for the test cases with dst precision f16, e.g., convert_f8_to_f16, it would end up with testing the functionality of convert_f8_to_f32, because of precision transformation here https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp#L399-L401.

Reply:
Why don't we do the same for bf16?
Shared test is not the best plave for such specific tuning. Better to update custom convresion single layer test in CPU plugin.

Reply:
Applied and added custom convresion single layer test in CPU plugin. Thanks Dmitry!